orb.yml

```
@@ -2892,6 +2918,46 @@ commands:
           background: << parameters.run_in_background >>
           name: << parameters.step_name >>
           command: yarn e2e:check-deps
+    
+  extract_cypress_artifacts:
+    steps:
+      - run:
+          name: Get cypress artifacts
+          working_directory: ~/logs
+          when: on_fail
+          command: |
+            # Getting the logs
+            set +e
+            docker cp code:/code/creator-app/packages/creator-app/cypress/videos ~/code/creator-app/packages/creator-app/cypress/videos
+            docker cp code:/code/creator-app/packages/creator-app/cypress/screenshots ~/code/creator-app/packages/creator-app/cypress/screenshots
+            set -e
+
+  extract_e2e_logs:
```

does this make sense @xavidop? since we ignore errors it should just capture logs for all the known services that actually ran as part of the job and ignore all the others